tests/integration/README.md

```diff
@@ -0,0 +1,388 @@
+    
+    # Switch selinux into permissive mode
+    sed -i "s/SELINUX=enforcing/SELINUX=permissive/" /etc/selinux/config
+    
+    # enable qemu-ga
+    systemctl enable qemu-guest-agent
+    
+    # install python pip
+    # for centos8 and ol8
+    curl https://bootstrap.pypa.io/pip/get-pip.py | python3
+    # for centos7 and ol7
+    # curl https://bootstrap.pypa.io/pip/2.7/get-pip.py | python
+    
+    # install rsync
+    yum install -y rsync
+    
+    # Remove installed version of convert2rhel
+    yum remove convert2rhel -y
+    
+    # exit the VM Ctrl+D
+
+# shutdown the vagrant VM
+virsh shutdown c2r-vagrant_default
+
+# clone the VM and give it corresponding name (--name)
+# SYSTEM=centos8 -> c2r_centos8_template
+# SYSTEM=centos7 -> c2r_centos7_template
+# SYSTEM=ol8 -> c2r_oracle8_template
+# SYSTEM=ol7 -> c2r_oracle7_template
+virt-clone --original c2r-vagrant_default --name c2r_centos8_template --auto-clone --check disk_size=off
+
+# remove the origin vagrant VM
+virsh undefine --domain c2r-vagrant_default
+
+# remove stale volume
+vagrant destroy -f
+
+# Now do the same for SYSTEM={centos7,ol8,ol7}
+```
+
+3. Perform post setup for each VM 
+   
+3.1. In Virtual Machine Manager go to Edit -> Preferences and set enable
+xml editing
+
+3.2. For each VM in VM Manager click Open and:
+- set network source (NIC device) to default (Virtual network 'default': NAT)
+- add qemu-ga channel, by clicking Add Hardware button:
+select Channel, use XML to declar the device and add:
+```xml
+<channel type="unix">
+  <target type="virtio" name="org.qemu.guest_agent.0"/>
+  <address type="virtio-serial" controller="0" bus="0" port="1"/>
+</channel>
+```
+click Finish, click Apply
+
+## Step 3 - Setup TMT-Pytest framework
+Install local development environment:
+```bash
+# cd to the cloned convert2rhel github repository dir, e.g.
+cd ~/Documents/convert2rhel/
+
+# install venv, python deps and build images
+rm .images .install
+make install
+
+# set secrets for running integration tests
+vim .env
+```
+
+# Creating integration tests
+
+We use TMT testing framework as a meta testing platform which manages
+integration tests lifecycle (https://tmt.readthedocs.io/en/stable/index.html).
+It uses FMF format (https://fmf.readthedocs.io/en/stable/index.html) to declare
+testing plans and tests.
+
+Basic structure of the feature integration test is as follows:
+```bash
+# feature plan is stored as a directory in plans/integration dir
+#     in our case this is - inhibit-if-oracle-system-uses-not-standard-kernel
+$ tree plans
+plans/
+├── integration
+│   └── inhibit-if-oracle-system-uses-not-standard-kernel
+│       ├── main.fmf
+│       ├── vm_oracle7.fmf
+│       └── vm_oracle8.fmf
+└── main.fmf
+# primary (root) plan
+$ cat plans/main.fmf
+execute:
+    how: tmt
+discover:
+    how: fmf
+prepare:
+    how: ansible
+    playbook:
+        - tests/ansible_collections/basic_setup.yml%      
```

Comment:
```suggestion
        - tests/ansible_collections/basic_setup.yml
```

Reply:
happy we'll have soon end-of-line fix hook in pre-commit:) Fixed

Reply:
fixed